manifests/h/HeroicGamesLauncher/HeroicGamesLauncher/1.8.1/HeroicGamesLauncher.HeroicGamesLauncher.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: HeroicGamesLauncher.HeroicGamesLauncher
+PackageVersion: 1.8.1
+PackageLocale: en-US
+Publisher: Flavio F
+PublisherUrl: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher
+PublisherSupportUrl: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/Support.md
+#PrivacyUrl: 
+Author: Flavio F
+PackageName: Heroic 1.8.1
```

```suggestion
PackageName: Heroic
```
You can exclude the version from the PackageName since it looks for `Heroic "wildcard"`